### PR TITLE
Adds Pagevamp and PeaceNepal

### PR DESCRIPTION
**Pull request by geshan**
- [x] PeaceNepal is a web shop in Kupondole
- [x] Pagevamp is a product company in Sanepa converting FB pages to websites

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,6 +66,8 @@ If you cannot provide a valid website for your company your PR will not be accep
 - [nLocate](https://nlocate.com) | Harihar Bhawan, Lalitpur | Locate services, businesses, events around you
 
 ## P - R
+- [PageVamp](https://www.pagevamp.com/) | Sanepa, Lalitpur | Create and Update Website from Facebook Page
+- [PeaceNepal.com](http://peacenepal.com.np/) | Kandevtastan, Kupondole, Lalitpur | Website and Software Development, SEO, Web Hosting, Web Graphic and Logo Design
```
Comment (mesaugat) [inline README.md]: Thanks @geshan dai. Can we narrow it down to `Kupondole, Lalitpur` to make it consistent.

**Comment by geshan** [comment]
ok now @mesaugat ?

**Comment by mesaugat** [comment]
@geshan One more thing. We should make their name `Peace Nepal DOT Com Pvt. Ltd.` as mentioned in this [page](http://peacenepal.com.np/contact-us.html) - `PeaceNepal.com` seems confusing. What do you say?

**Comment by geshan** [comment]
done.

**Comment by mesaugat** [comment]
Thanks 🎉

**Pull request merged**